endpoint.go

```diff
@@ -600,7 +606,23 @@ func (ep *endpoint) sbLeave(sbox Sandbox, options ...EndpointOption) error {
 	return sb.clearDefaultGW()
 }
 
-func (ep *endpoint) Delete() error {
+func (n *network) validateForceDelete(locator string) error {
+	if n.Scope() == datastore.LocalScope {
+		return nil
+	}
+
+	if locator == "" {
+		return fmt.Errorf("invalid endpoint locator identifier")
+	}
+
+	if n.getController().isNodeAlive(locator) {
```

Comment:
I disagree with this change, force is force.

In engine you can `docker rmi -f <image>` even if the engine has containers using this image. 
I agree it can lead to inconsistent states, but the user did force it.